directus/src/update-notifier.ts

```diff
@@ -0,0 +1,82 @@
+import boxen, { type Options as BoxenOptions } from 'boxen';
```

Comment:
Lets turn this file into the `update-check` package 🙂 I don't think it should be part of the `directus` top-level directly 👍🏻

Reply:
I knew it 😆 👍